.changeset/pink-bears-refuse.md

```diff
@@ -0,0 +1,5 @@
+---
+"@salt-ds/core": patch
+---
+
+Fixed ComboBox having a button when it has nothing to show.
```

Comment:
"a button" can be ambiguous ?

Reply:
Any recommendation? "Trigger button", "button adornment"